packages/client/src/config.ts

```diff
@@ -1,6 +1,38 @@
 import { SetupContractConfig } from "@latticexyz/std-client";
 import { Wallet } from "ethers";
+import * as wagmiChains from "@wagmi/chains";
+
+// TODO: move this to a chains/config package
+const latticeTestnet = {
+  name: "Lattice Testnet",
+  id: 4242,
+  network: "lattice-testnet",
+  nativeCurrency: { decimals: 18, name: "Ether", symbol: "ETH" },
+  rpcUrls: {
+    default: {
+      http: ["https://follower.testnet-chain.linfra.xyz"],
+      webSocket: ["wss://follower.testnet-chain.linfra.xyz"],
+    },
+    public: {
+      http: ["https://follower.testnet-chain.linfra.xyz"],
+      webSocket: ["wss://follower.testnet-chain.linfra.xyz"],
+    },
+  },
+} satisfies wagmiChains.Chain;
+
+const chains = { ...wagmiChains, latticeTestnet };
```

Comment:
we could move this + a re-export of wagmi's chain object indexed by chain id from a new mud package (or some other general purpose one)